src/main/java/pl/simpleascoding/tutoringplatform/domain/advertisement/AdvertisementCategory.java

```diff
@@ -0,0 +1,5 @@
+package pl.simpleascoding.tutoringplatform.domain.advertisement;
+
+public enum AdvertisementCategory {
```

Comment:
Documentation here would be greatly appreciated:
- for what purposes this enum is created
- what the individual values mean